go-fuzz/main.go

```diff
@@ -88,5 +104,36 @@ func main() {
 		go slaveMain()
 	}
 
+	if *flagMasterStats != "" {
```

Comment:
I feel that these handlers should be defined on Master and moved to master.go, along with writerSet and broadcastStats.
That would reduce number of global variables, and make it easier to export more stats (e.g. if there is a new crasher, you probably want to look at output and download the file remotely as well).